### ISSUE TITLE
API_URL is missing when compile template.

### ISSUE THREAD

**Issue by pyaehein** [nuxt-community/recaptcha-module#22]
I got an error like this.

```
Error: Could not compile template /MyProject/node_modules/@nuxtjs/recaptcha/lib/plugin.js: API_URL is not defined
```

But it can be fix like this.

```
if (this.version === 2) {
  script.setAttribute('src', API_URL + '?hl=' + this.language)
} else {
  script.setAttribute('src', API_URL + '?render=' + this.siteKey)
}
```

**Comment by mvrlin** [comment]
Hello, what version of Nuxt are you using?

**Comment by pyaehein** [comment]
I am currently using old version of nuxt js ("nuxt": "^1.4.1")

**Comment by mvrlin** [comment]
@pyaehein fixed in the **0.5.2** release 😊
